cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -138,6 +140,18 @@ Connection:
 
      When connecting over TCP, the -instances parameter is required.
 
+     Supplying INSTANCES environment variable achieves the same effect.  One can
+     use that to keep k8s manifest files constant across multiple environments.
+     Supplying both INSTANCES and -instances will result in joining of two inputs.
```

Comment:
I would suggest let the "-instances" flag overrides the "INSTANCES" env if both specified (which in my opinion shouldn't).  If somebody specify by flag, one might not want it to be affected by the env variable.

Reply:
@AthenaShi that was the original goal, I changed it to this based on @Carrotman42 's note.  Will revert if this indeed desired.

Reply:
@AthenaShi can't find that comment on github (miss gerrit), pasting it from my email,

> Kevin Malachowski <notifications@github.com> 
> How about appending the two instead of over writing? It'd be in the same spirit as how -instances and -instance_metadata can be used together.

Reply:
done.